flake.nix

```
@@ -51,7 +51,7 @@
 
               buildInputs = optionals pkgs.stdenv.isDarwin [
                 pkgs.darwin.apple_sdk.frameworks.CoreServices
-              ];
+              ] ++ [pkgs.iconv];
```

Can you test if this diff would also work? I think it would be more consistent with the dev shell output's build inputs.
```suggestion
                pkgs.libiconv
              ];
```

---

Yeah, it does work as well